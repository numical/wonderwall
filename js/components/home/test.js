'use strict';
/* eslint-env jasmine */

import React from 'react';
import { Link } from 'react-router';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import HomePage from './index.js';

describe('Home Page ', () => {
  let page;

  beforeAll(() => {
    page = shallow(<HomePage />);
  });

  it('should match previous snapshot', () => {
    expect(shallowToJson(page)).toMatchSnapshot();
  });

  it('should say Hello world!', () => {
    expect(page.find('.page-title').text()).toEqual('Hello World!');
  });

  it('should have a Link to config page', () => {
    expect(page.find(Link).prop('to')).toEqual('/config');
  });
});

