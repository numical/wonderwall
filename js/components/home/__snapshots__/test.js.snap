exports[`Home Page  should match previous snapshot 1`] = `
<div>
  <div
    className="page-title">
    Hello World!
  </div>
  <Link
    onlyActiveOnIndex={false}
    style={Object {}}
    to="/config">
    Configure
  </Link>
</div>
`;
